main.py

```
@@ -15,59 +15,62 @@
 from modules.primary_translators.reverso_translator import ReversoTranslationService
 from modules.primary_translators.translatecom_translator import TranslatecomTranslationService
 from modules.primary_translators.yandex_translator import YandexTranslationService
+from modules.primary_translators.dummynone_translator import DummyNonePrimaryTranslationService
 from modules.secondary_translators.ollama_translator import OllamaTranslationService
```

Ruff (0.8.2)

`modules.secondary_translators.ollama_translator.OllamaTranslationService` imported but unused

Remove unused import: `modules.secondary_translators.ollama_translator.OllamaTranslationService`

(F401)